pipelines/resource_pipeline.go

```diff
@@ -22,6 +22,7 @@ const DefaultTimeout = 20 * time.Minute
 // 1. Pipeline clusters include a `Label` field.
 // 2. Spark version is not required (and shouldn't be specified) for pipeline clusters.
 // 3. num_workers is optional, and there is no single-node support for pipelines clusters.
+// When adding new attributes, also modify the isEmptyClusterStruct function below
```

Comment:
have you tried `reflect.IsZero` way?

https://www.geeksforgeeks.org/reflect-iszero-function-in-golang-with-examples/

Reply:
I looked to reflect, but read that it may not handle pointers, etc. correctly

Reply:
I’ll look Monday

Reply:
i think it's more of an issue in `slice_set`, btw. e.g. diff is not properly picked up

Reply:
agree, but this requires more time to investigation. I had a hope to have quick & dirty fix, and then fix correctly when get time